astroquery/simbad/core.py

```diff
@@ -217,13 +217,14 @@ def list_wildcards():
     def list_votable_fields(self):
         """List all options to add columns to SIMBAD's output.
 
-        They are of three types:
+        They are of four types:
 
         - "column of basic": a column of the basic table. There fields can also be explored with
           `~astroquery.simbad.SimbadClass.list_columns`.
         - "table": a table other than basic that has a declared direct join
         - "bundle of basic columns": a pre-selected bundle of columns of basic. Ex: "parallax" will add all
           columns relevant to parallax
+        - "filter": a filter name
```

Comment:
maybe rephrase the description to make sure it's clear that `filter` is for an optical filter rather than a VO/any other computing filtering expression. I expected the latter :)

(maybe if possible, then rename it to `filter name`? that would make it unambiguous)

Reply:
Ah sure! Did not think about that ^^
This is renamed (and I also tried to make it a bit clearer in the documentation)